cfpb/consumerfinance.gov
# Create `m-full-width-media` modifier and caption element

The `m-inset` element and modifier weren't contained in an `m-inset` block. This moves those pieces to `m-full-width-media`.

## Changes

- Create `m-full-width-media` modifier and caption element


## How to test this PR

1. `yarn build` and pages full width images with a bottom border be unchanged from production http://localhost:8000/about-us/blog/common-errors-credit-report-and-how-get-them-fixed/ and pages with a caption should be unchanged http://localhost:8000/about-us/blog/initial-fresh-start-program-changes-followed-by-increased-credit-scores-for-affected-borrowers/
